clientv3/client.go

```diff
@@ -200,5 +200,6 @@ func dialEndpointList(c *Client) (*grpc.ClientConn, error) {
 }
 
 func isRPCError(err error) bool {
-	return grpc.Code(err) != codes.Unknown
+	return grpc.Code(err) != codes.Unknown ||
+		err == grpc.ErrClientConnClosing
```

Comment:
If the current client conn is closing, should'nt we retry? (when we meet rpc error, we will not retry)

We are not sure about if just the conn is closing or the entire client is closing, right? I feel we should only exit when the client ctx is cancelled.